Misc/NEWS.d/next/Library/2022-09-17-13-15-10.gh-issue-96819.6RfqM7.rst

```diff
@@ -0,0 +1 @@
+Fixed the block to guarantee that the length of a write to a pipe is not greater than PIPE_BUF
```

Comment:
```suggestion
Fixed the check that guarantees that the length of a write to a pipe is not greater than PIPE_BUF
```

Reply:
Thank you for correcting my English! Your suggestion seems to be great.

Reply:
No worries! I struggle with writing good and short changelog entries myself many times! 😄